### ISSUE TITLE
Link in message missing link accessibillity label

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1453]
### Description of problem
In messages that are sent from a contact that contains a link, the link is missing the "link" accessibility label/role. When focused, the link is read out but a non-sighted person will not know if it's a link they can tap.

### Expected behavior
Should contain the link accessibility role/label. It normally reads out as "[text of hyperlink]...Link"

### Steps to reproduce
- Send a message with a link to a contact
- turn on voice over
- swipe over to the message with a link

### Environment
- [X] Occurs on Android
- [X] Occurs on iOS

Build #: 1193

Android Device Model: OnePlus 6

iOS Device Model: iPhone SE

### Severity
- [ ] High
- [ ] Medium
- [X] Low

**Comment by bryce-mcmath** [comment]
The accessibilityRole of link in nested `Text` elements is not picked up on iOS until RN 0.69, so we'll have to wait until after our React Native upgrade to fix this one

**Comment by wadeking98** [comment]
resolved by: https://github.com/hyperledger/aries-mobile-agent-react-native/pull/998

**Comment by jleach** [comment]
Tested on 1.0.12 1325 and I don't see any accessibility labels on the link in chat:

![IMG_9D0F150ADF20-1](https://github.com/bcgov/bc-wallet-mobile/assets/390891/509915b4-051d-406f-b12c-9d52902ca4ab)

**Comment by bryce-mcmath** [comment]
Until we start using the new RN architecture (with Fabric) inline nested links will not have accessibility features. This can be avoided in the interim by using the Link component *outside* of the text surrounding it. It's not an elegant solution but it's the only one that works for now

**Comment by jleach** [comment]
This may require a re-think. Moving out of this sprint to est ready.
